src/components/AddFolderButton.jsx

```diff
@@ -1,10 +1,6 @@
-// // AddFolderButton -> '폴더 추가 +'가 들어가는 부분(기능은 추후 구현)
-// function AddFolderButton() {
-//   return (
-//     <div>
-//       <h2>폴더추가버튼 자리입니다.</h2>
-//     </div>
-//   );
-// }
+// AddFolderButton -> '폴더 추가 +'가 들어가는 부분(기능은 추후 구현)
+function AddFolderButton({ className }) {
+  return <button className={className}>폴더추가 +</button>;
```

Comment:
P2:
classname을 왜 받아야 하는건가요? 폴더추가 버튼이 일정한 디자인이 있기 때문에 따로 컴포넌트로 빼신 거 아닌가요?
만약 일정한 디자인이 있는데 화면 사이즈에 따라 변경되는거라면 css로 처리하시면 좋을 것 같아요.

```suggestion
  return <button className={`${styleModule.addFolderButton} ${classname}`} type='button'>폴더추가 +</button>;
```